### DISCUSSION TITLE
time code format in custom format template

### DISCUSSION THREAD

**Discussion by faon-92** [SubtitleEdit/subtitleedit#8659] [Q&A]
Hi everyone,

I'm wondering how get time code format as hh:mm:ss:ff, inside the custom text format template (see screen capture). It's under the file-export-export custom text format...

thanks in advance,
![SubtitleEdit_vnXU19vxvb](https://github.com/user-attachments/assets/9f58b740-1cc8-479d-b45e-c9465e73a806)

**Comment by PreetM07** [comment]
jsut put your cursor after . backspace to remove and add ":" from keyboard...!
